src/api/scrapper/news.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC
```

Comment:
use this to solve this comment: https://github.com/soniasonia/simple-page/pull/1/files#r562878296

but move this file to `./src/api/data/article.py` instead

Reply:
OK